### ISSUE TITLE
Link Preview does not work while Beat Link Trigger is running

### ISSUE THREAD

**Issue by boadnb** [Deep-Symmetry/beat-link-trigger#182]
When Beat Link Trigger is open on a PC on the network, Link Preview does not work.

Gear:
3X CDJ3000
DJM A9
Netgear GS308

To Reproduce:
Power on devices
Once link connection established, open Beat Link Trigger
Enable Link Cue on the DJM A9
Utilize preview feature in the browser on CDJ3000 or by holding down on waveform in playback window.
No audio is heard, although play marker advances (showing that cdj3000 is indeed trying to play audio).

If you close beat link trigger and try again, Link Preview works normally. Once you open Beat Link Trigger, Link Preview stops working.

Issue appears to be related to Beat Link Trigger.

**Comment by brunchboy** [comment]
Interesting. I have never tried to use this feature. I happen to have an A9 and a couple of CDJ-3000s set up right now, can you give me more details about how this feature is enabled and how it is supposed to work?

When you are using BLT, are you using a real player number or not? If you can capture the relevant network traffic, figure out what is going on, and submit a pull request to fix the issue, it would be much appreciated! The analysis of what we know about the protocol is found [here](https://djl-analysis.deepsymmetry.org/djl-analysis/packets.html), and the library that BLT uses to interact with the players is [here](https://github.com/Deep-Symmetry/beat-link/tree/main). However you should note that the problem may be in the mixer and/or CDJ firmware, in which case we would not be able to fix it, and the only recourse would be to not try to use these features together.

**Comment by brunchboy** [comment]
I am guessing the feature you are describing is the same as [Touch Audio](https://djl-analysis.deepsymmetry.org/djl-analysis/touch_audio.html#audio-data) which was documented by @nudge. So those are the packets you are going to want to be studying in Wireshark, to see what is different between the cases when BLT is running and when it is not.

**Comment by brunchboy** [comment]
I figured out how to use the Link Cue feature on the mixer, and it continues to work fine even when I have Beat Link Trigger up and running and displaying the status of the players. So there must be something unusual going on in your situation. Without being able to reproduce the problem, I’m not going to be able to help diagnosing or solving it, I’m afraid.

After a bit more testing, it does not matter whether BLT is using a real player number (player 1 in this test) or the default of player 7. And Link Cue works fine on both the headphones A and B jacks.

**Comment by brunchboy** [comment]
Are you playing tracks from rekordbox, or a USB? My tests are from a USB, I don’t have spare computers handy most of the time to try running rekordbox and BLT at the same time.

**Comment by boadnb** [comment]
Hello,

Playing from USB. Sorry for the delay. So, the issue lays with my A9. I did
a bit of playing around. I swapped in the v10 and everything works as
intended every single time.
After putting the a9 back in, I played with startup routines. If I power on
the CEO’s, then power on the a9 after they connect to the network,
everything works as intended. I was also able to reproduce the issue
without running beat link trigger by powering on the a9, then powering on
the CDj’s after the a9 fully powered on. I have submitted a ticket to
Pioneer, now, as swapping cdjs out for other gear during rentals isn’t
uncommon.

Thank you
